app/connectors/BtaNavBarPartialConnector.scala

```diff
@@ -17,24 +17,34 @@
 package connectors
 
 import config.FrontendAppConfig
+
 import javax.inject.{Inject, Singleton}
 import models.btaNavBar.NavContent
 import play.api.Logger
-import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReadsInstances}
+import play.api.libs.json.{Json, Reads}
+import uk.gov.hmrc.http.client.HttpClientV2
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpReadsInstances, HttpResponse}
 
+import java.net.URL
 import scala.concurrent.{ExecutionContext, Future}
 
 
 @Singleton
-class BtaNavBarPartialConnector @Inject()(val http: HttpClient,
+class BtaNavBarPartialConnector @Inject()(val http: HttpClientV2,
                                           val config: FrontendAppConfig) extends HttpReadsInstances {
 
-  lazy val btaNavLinksUrl: String = config.btaService + "/business-account/partial/nav-links"
+  lazy val btaNavLinksUrl: URL = new URL(config.btaService + "/business-account/partial/nav-links")
```

Comment:
_usage is deprecated => see below: 
Deprecated
Use URI. toURL() to construct an instance of URL. See the note on constructor deprecation for more details._